source/about-mongosync.txt

```diff
@@ -157,6 +157,73 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization.
+
+Consistency 
+~~~~~~~~~~~
+
+For the destination cluster to have an up to date, consistent view of the 
+source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+If the source cluster shuts down before ``mongosync`` can commit, such as in 
+a disaster scenario, the destination cluster might not have a consistent 
+snapshot of the source data, and transactions might not appear atomically on 
+the destination cluster. This can occur because ``mongosync`` temporarily alters 
+some collection characteristics during synchronization. The original values are 
+restored during the commit process.
+
+.. list-table::
```

Comment:
I think we need a table header here saying something like "Temporarily changed characteristics during a sync" or something